wallaby.js

```diff
@@ -0,0 +1,64 @@
+const babelConfig = require("./build-settings/babel.config.js");
+
+module.exports = function (wallaby) {
+    return {
+        files: [
+            'packages/wonder-blocks-*/**/*.js',
+            'package.json', // <--
+            { pattern: 'config/jest/*.js', instrument: false },
+            'shared-unpackaged/**/*.js',
+            'utils/**/*.js',
+            'packages/wonder-blocks-*/package.json',
+            { pattern: 'packages/wonder-blocks-*/dist/**/*.js', instrument: false },
+            '!packages/wonder-blocks-*/**/*.test.js',
+            '!packages/wonder-blocks-*/node_modules/**',
+            '!shared-unpackaged/**/*.test.js',
+        ],
+
+        filesWithNoCoverageCalculated: [
+            'config/jest/*.js',
+            'packages/wonder-blocks-*/dist/**/*.js',
+            'utils/**/*.js',
+        ],
+
+        tests: [
+            'packages/wonder-blocks-*/**/*.test.js',
+            'shared-unpackaged/**/*.test.js',
+        ],
+
+        compilers: {
+            "packages/wonder-blocks-*/**/*.js": wallaby.compilers.babel(babelConfig),
+            "shared-unpackaged/**/*.js": wallaby.compilers.babel(babelConfig),
+            "utils/**/*.js": wallaby.compilers.babel(babelConfig),
```

Comment:
Even though we're using jest normally jest would be responsible for compiling files, wallaby compiles files itself before passing them for testing.  The reason why wallaby handles the compilation is that some its features like advanced logging and live comments requires instrumenting the code. https://wallabyjs.com/docs/intro/advanced-logging.html

Reply:
Make sense although I wonder how much time we'd end up in a situation where a test fails because of a wallaby compilation issue?

Reply:
Does this mean wallaby will run on all of wonderblocks?

Reply:
If things are wonky we can always redirect imports to the dist files instead of the source files.

Yes, it runs all of the tests and produces nice coverage reports with rollups in the browser app.  wallabyjs.com/app redirects to localhost:54321 (I forget the exact port).

<img width="1642" alt="Screen Shot 2019-05-15 at 5 04 18 PM" src="https://user-images.githubusercontent.com/1044413/57809467-83377d00-7733-11e9-9ebb-9252b6c79baf.png">